stream-chat-android/src/test/java/com/getstream/sdk/chat/Mother.kt

```diff
@@ -136,9 +136,10 @@ internal fun createAttachment(
     fallback: String? = randomString(),
     uploadFile: File? = null,
     uploadState: Attachment.UploadState? = null,
-    extraData: MutableMap<String, Any> = mutableMapOf()
+    extraData: MutableMap<String, Any> = mutableMapOf(),
+    authorLink: String? = randomString(),
 ): Attachment = Attachment(
-    authorName, titleLink, thumbUrl, imageUrl, assetUrl, ogUrl, mimeType,
```

Comment:
It would be nice to start using named parameters here so we don't accidentally pass them in wrong order